### DISCUSSION TITLE
v4.8.1-rc.1

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5715] [Announcements]
Release candidate for 4.8.1 

## Fixed

Uses the correct hash for `*.html` files which had CSP errors in 4.8.0. 
See https://github.com/coder/code-server/pull/5712 for details. 

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.8.1-rc.1'>v4.8.1-rc.1</a>.</em>

**Comment by jeengbe** [comment]
When will you release this? (to Docker Hub)

**Reply by jsjoeio** [reply]
Hmm...I don't think we have the tooling/workflow setup to publish release candidates to Docker Hub at the moment but we will publish 4.8.1 tomorrow as long as there are no issues. cc @code-asher in case he knows something

**Reply by jeengbe** [reply]
Great, thanks for the quick reply! :)

**Reply by code-asher** [reply]
My view on things is that we want to keep release candidates contained to GitHub but we can revisit that at some point once the release candidate workflow stabilizes.

**Reply by jsjoeio** [reply]
We can revisit this and tag you when that discussion happens so you can share your thoughts too (no promises but would welcome your input!).

**Reply by jeengbe** [reply]
I'm totally fine with waiting for a stable release on Docker Hub (as it is very easily possible to switch pack to the previous version).
